app/src/global.d.ts

```diff
@@ -6,6 +6,6 @@ declare module '@realitio/realitio-lib/formatters/template'
 
 declare type Maybe<T> = T | null
 
-declare type KnownToken = 'dai' | 'usdc'
+declare type KnownToken = 'sai' | 'usdc' | 'dai' | 'weth' | 'owl'
```

Comment:
I believe the `sai`/`dai` distinction is just temporary to distinguish between the old and the new token. The app should continue to call it `dai`, and there shouldn't be a `sai` token.

@leolower is this correct?

Reply:
Besides, this is missing `cDai`.

Reply:
Correct, we should replace SAI with DAI. NO need to support both.

Reply:
I remove SAI, and added cDAI :+1: